packages/ppx/test/css-support/non-valid-property-location-error-nested.t/run.t

```diff
@@ -0,0 +1,55 @@
+This test ensures the ppx generates the correct output against styled-ppx.css_native
+If this test fail means that Css_Js_Core or CssJs_Legacy_Core (from styled-ppx.css or styled-ppx.css_native) are not in sync with the ppx
+
+This test only runs against Css_Js_Core from styled-ppx.css_native
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.10)
+  > EOF
+
+  $ cat > dune << EOF
+  > (executable
+  >  (name input)
+  >  (libraries styled-ppx.emotion_native styled-ppx.css_native)
+  >  (preprocess (pps styled-ppx)))
+  > EOF
+
+  $ dune build
+  File "input.re", lines 5-6, characters 13-13:
```

Comment:
are columns correct here?

Reply:
yup all correct now

Reply:
seems like characters 13-13 is wrong thought

Reply:
yeah, this is because the declaration name loc_start is off from the parser. i've fixed it by changing it based on the loc_end and the property string length